internal/pubsub/pubsub.go

```diff
@@ -88,14 +91,23 @@ func (t *Topic) Send(ctx context.Context, m *Message) error {
 	return t.batcher.Add(ctx, m)
 }
 
-// Close flushes pending message sends and disconnects the Topic.
+// Shutdown flushes pending message sends and disconnects the Topic.
 // It only returns after all pending messages have been sent.
-func (t *Topic) Close() error {
+func (t *Topic) Shutdown(ctx context.Context) error {
 	t.mu.Lock()
 	t.err = errors.New("pubsub: Topic closed")
 	t.mu.Unlock()
-	t.batcher.Shutdown()
-	return nil
+	c := make(chan struct{})
```

Comment:
Same here, this isn't related to the rename is it?

Reply:
Same answer.